update CFR homepage (#6194)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/activity-reports/activity-reports.md

  
## Hide user details in the reports

> Reports provide information about your organization’s usage data. By default, reports display information with identifiable names for users, groups, and sites. Starting September 1st 2021, we are hiding user information by default for all reports as part of our ongoing commitment to help companies support their local Privacy laws.

Reviewer: ```suggestion
Reports provide information about your organization’s usage data. By default, reports display information with identifiable names for users, groups, and sites. Starting September 1, 2021, we are hiding user information by default for all reports as part of our ongoing commitment to help companies support their local privacy laws.
```

  
Your user list will look like this:
  
![Reports - anonymized user list.](../../media/2ed99bce-4978-4ee3-9ea2-4a8db26eef02.png)
  

> Global administrators can revert this change for their tenant and show identifiable user information if their organizations privacy practices allow. This can be achieved in the admin center with simple steps. 

Reviewer: ```suggestion
Global administrators can revert this change for their tenant and show identifiable user information if their organization's privacy practices allow it. It can be achieved in the Microsoft 365 admin center by following these steps:
```

  
1. In the admin center, go to the **Settings** \> **Org Settings** \> **Services** page.

2. Select **Reports**. 
  

> 3. Under **choose how to show user information**, select the options you want, and then save your changes.  

Reviewer: ```suggestion
3. Under **Choose how to show user information**, select the options you want, and then save your changes.  
```

> It'll take a few minutes for these changes to take effect on the reports in the reports dashboard. This setting also applies to the reports API. Showing identifiable user information is a logged event in the Microsoft 365 Compliance Center Audit log.   

Reviewer: ```suggestion
It'll take a few minutes for these changes to take effect on the reports in the reports dashboard. This setting also applies to the reports API. Showing identifiable user information is a logged event in the Microsoft 365 compliance center audit log.   
```
